src/coreclr/vm/appdomain.cpp

```diff
@@ -1373,6 +1373,9 @@ void SystemDomain::LoadBaseSystemClasses()
     // We have delayed allocation of CoreLib's static handles until we load the object class
     CoreLibBinder::GetModule()->AllocateRegularStaticHandles(DefaultDomain());
 
+    // Int32 has to be loaded first to break cycle in IShiftOperators
+    CoreLibBinder::LoadPrimitiveType(ELEMENT_TYPE_I4);
```

Comment:
What's this change?

Reply:
A change from Jan that fixes a VM bug that was preventing this change from working.